awx/ui_next/src/screens/Inventory/InventoryGroups/InventoryGroupDetail/InventoryGroupDetail.jsx

```diff
@@ -0,0 +1,157 @@
+import React, { useState } from 'react';
+import { t } from '@lingui/macro';
+
+import { CardBody, Button } from '@patternfly/react-core';
+import { withI18n } from '@lingui/react';
+import { withRouter, Link } from 'react-router-dom';
+import styled from 'styled-components';
+import { VariablesInput as CodeMirrorInput } from '@components/CodeMirrorInput';
+import ErrorDetail from '@components/ErrorDetail';
+import AlertModal from '@components/AlertModal';
+import { formatDateString } from '@util/dates';
+
+import { GroupsAPI } from '@api';
+import { DetailList, Detail } from '@components/DetailList';
+
+const VariablesInput = styled(CodeMirrorInput)`
+  .pf-c-form__label {
+    font-weight: 600;
+    font-size: 16px;
+  }
+  margin: 20px 0;
+`;
+const ActionButtonWrapper = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  margin-top: 20px;
+  & > :not(:first-child) {
+    margin-left: 20px;
+  }
+`;
+function InventoryGroupDetail({ i18n, history, match, inventoryGroup }) {
+  const {
+    summary_fields: { created_by, modified_by },
+  } = inventoryGroup;
+  const [error, setError] = useState(false);
+  const [isDeleteModalOpen, setIsDeleteModalOpen] = useState(false);
+
+  const handleDelete = async () => {
+    setIsDeleteModalOpen(false);
+    try {
+      await GroupsAPI.destroy(inventoryGroup.id);
+      history.push(`/inventories/inventory/${match.params.id}/groups`);
+    } catch (err) {
+      setError(err);
+    }
+  };
+
+  return (
+    <CardBody style={{ paddingTop: '20px' }}>
```

Comment:
I prefer `css=` for these. It comes with our `styled-components` library and lets you write these like you actually would in a stylesheet instead of with js objects. This one could be `css="padding-top: 20px"`